server/src/test/java/org/opensearch/discovery/PeerFinderTests.java

```diff
@@ -336,16 +336,16 @@ public void testDoesNotAddUnreachableNodesFromUnicastHostsList() {
     }
 
     public void testDoesNotAddNonMasterEligibleNodesFromUnicastHostsList() {
```

Comment:
`testDoesNotAddNonMasterEligibleNodesFromUnicastHostsList ` -> `testDoesNotAddNonClusterManagerEligibleNodesFromUnicastHostsList `

Reply:
Changed the method name. 👍